src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/endpoints/JurisdictionEndpoint.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.endpoints;
+
+import feign.FeignException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.actuate.endpoint.annotation.Endpoint;
+import org.springframework.boot.actuate.endpoint.annotation.ReadOperation;
+import org.springframework.boot.actuate.endpoint.annotation.Selector;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam.Credential;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam.JurisdictionToUserMapping;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam.NoUserConfiguredException;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static java.util.Map.Entry;
+
+@Component
+@Endpoint(id = "jurisdictions")
+@EnableConfigurationProperties(JurisdictionToUserMapping.class)
+public class JurisdictionEndpoint {
```

Comment:
name not clear. this endpoint does not return jurisdictions.

Reply:
Why not a controller?

Reply:
This is configuration based rest endpoint. Since people have trouble with existing ones and I have a pretty strong position of **not** including it into health indicator list, it is beneficial to use custom actuator endpoint to achieve what this ticket has defined

Reply:
I'm not suggesting a custom health indicator, just a controller.
Since we've configured the base path for management endpoints to be `/` I think there's no difference between a controller and `@Endpoint`.